components/Form.tsx

```diff
@@ -46,6 +46,9 @@ export default function Form({ title, questions, listingId, includeEventsAttende
   const [formData, setFormData] = useState<FormData>(initialValues);
   const [resumeFileName, setResumeFileName] = useState<String>("");
   const [imageFileName, setImageFileName] = useState<String>("");
+  const [resumeFileSize, setResumeFileSize] = useState<number>(0);
+  const [imageFileSize, setImageFileSize] = useState<number>(0);
+  const MAX_FILE_SIZE = 6
```

Comment:
What do you think about renaming MAX_FILE_SIZE to MAX_FILE_SIZE_BYTES and use the direct bytes to represent the value like 6 * 1024 * 1024 - 1 to represent 6 bytes * 1024 --> kb * 1024 --> mb minus one to handle file limit?

Reply:
I think it's more descriptive and readable to show the direct conversion too!

Reply:
Ooh yes i like that, but do you know when it should be 1MB = 1000*1000 Bytes vs 1MB = 1024*1024 Bytes? I saw some mixed answers online

Reply:
As we discussed before, if you see a storage capacity specified in megabytes (MB) without any indication, it's likely following the decimal system (1000 bytes per kilobyte). If you see it specified as mebibytes (MiB), it's following the binary system (1024 bytes per kibibyte)!